core/rfb.js

```diff
@@ -324,63 +339,55 @@ RFB.prototype = {
             RFB.messages.QEMUExtendedKeyEvent(this._sock, keysym, down, scancode);
         } else {
             if (!keysym) {
-                return false;
+                return;
             }
             Log.Info("Sending keysym (" + (down ? "down" : "up") + "): " + keysym);
             RFB.messages.keyEvent(this._sock, keysym, down ? 1 : 0);
         }
-
-        return true;
     },
 
     clipboardPasteFrom: function (text) {
-        if (this._rfb_connection_state !== 'connected' || this._view_only) { return; }
+        if (this._rfb_connection_state !== 'connected' || this._viewOnly) { return; }
         RFB.messages.clientCutText(this._sock, text);
     },
 
+    autoscale: function (width, height) {
+        if (this._rfb_connection_state !== 'connected') { return; }
+        this._display.autoscale(width, height);
+    },
+
+    viewportChangeSize: function(width, height) {
+        if (this._rfb_connection_state !== 'connected') { return; }
+        this._display.viewportChangeSize(width, height);
+    },
+
     // Requests a change of remote desktop size. This message is an extension
     // and may only be sent if we have received an ExtendedDesktopSize message
     requestDesktopSize: function (width, height) {
         if (this._rfb_connection_state !== 'connected' ||
-            this._view_only) {
-            return false;
+            this._viewOnly) {
+            return;
         }
 
-        if (this._supportsSetDesktopSize) {
-            RFB.messages.setDesktopSize(this._sock, width, height,
-                                        this._screen_id, this._screen_flags);
-            this._sock.flush();
-            return true;
-        } else {
-            return false;
+        if (!this._supportsSetDesktopSize) {
+            return;
         }
+
+        RFB.messages.setDesktopSize(this._sock, width, height,
+                                    this._screen_id, this._screen_flags);
```

Comment:
We did a cleanup in 89d2837fa8e3f0498bbac7b7194f453d88bcbc1f but apparently we missed this one.

Reply:
`Base64` is the only one not used. But it's not used on `master` either, so feel free to remove it there.

`KeyTable` is only used by `sendCtrlAltDel()`, which I considered removing since it is just a wrapper around `sendKey()`. Does anyone of you want to keep it?